parastell/source_mesh.py

```diff
@@ -437,14 +458,18 @@ def create_mesh(self):
         self.mbc.add_entity(self.mesh_set, self.verts)
 
         for phi_idx in range(self.num_phi - 1):
+            # Set alternation flag to true at beginning of each toroidal block
+            self.alt_flag = True
```

Comment:
Let's define this as a local variable and pass into the methods.

Reply:
a different name maybe useful.... Some algorithms use the nomenclature "red/black"